docs/antrea-network-policy.md

```diff
@@ -81,37 +81,41 @@ Tiers have the following characteristics:
+corresponding to the static tiers for default consumption, as well as a
+Baseline Tier that will be enforced after developer-created K8s NetworkPolicies.
+The details for these tiers are shown below:
 
 ```
-    Emergency -> Tier name "emergency" with priority "5"
+    Emergency   -> Tier name "emergency" with priority "5"
     SecurityOps -> Tier name "securityops" with priority "50"
-    NetworkOps -> Tier name "networkops" with priority "100"
-    Platform -> Tier name "platform" with priority "150"
+    NetworkOps  -> Tier name "networkops" with priority "100"
+    Platform    -> Tier name "platform" with priority "150"
     Application -> Tier name "application" with priority "250"
+    Baseline    -> Tier name "baseline" with priority "253"
 ```
 
-Any Antrea policy CRD referencing a static tier in its spec will now internally
-reference the corresponding Tier resource, thus maintaining the order of
-enforcement.
+Any Antrea-native policy CRD referencing a static tier in its spec will now internally
+reference the corresponding Tier resource, thus maintaining the order of enforcement.
 
-Previously, the static tiers created were as follows in the relative order of
-precedence:
+The static tier resources are created as follows in the relative order of
+precedence compared to K8s NetworkPolicies:
 
 ```
-    Emergency > SecurityOps > NetworkOps > Platform > Application  
+    Emergency > SecurityOps > NetworkOps > Platform > Application > K8s NetworkPolicy > Baseline
 ```
 
-Thus, all Antrea Policy resources associated with "Emergency" tier will be
-enforced before any other Antrea Policy resource associated with any other
+Thus, all Antrea-native Policy resources associated with "Emergency" tier will be
+enforced before any other Antrea-native Policy resource associated with any other
 tier, until a match occurs, in which case the policy rule's `action` will be
-applied. The "Application" tier carries the lowest precedence, and any Antrea
-Policy resource without a `tier` name set in its spec will be associated with
-the "Application" tier. Even though the policies associated with the
-"Application" tier carry the lowest precedence amongst all the tiers, they are
-still enforced before K8s NetworkPolicies. Thus, admin-created tiered Antrea
-Policy CRDs have a higher precedence than developer-created K8s
-NetworkPolicies.
+applied. **Any Antrea-native Policy resource without a `tier` name set in its spec
+will be associated with the "Application" tier.** Policies associated with the
+first five static, read-only tiers, as well as all custom tiers created with tier
+priority set under 250 (highest allowed), will be enforced before K8s NetworkPolicies.
+Antrea-native Polciies created in the Baseline Tier, on the other hand, will have
+lower precedence compared to developer-created K8s NetworkPolicies, which comes in
+handy when administrators want to enforced baseline policies like default-deny
```

Comment:
maybe the example should just be "default deny", at least until we have better API support for this one

Reply:
any thoughts on this, I saw you haven't changed it?

Reply:
Oh I though you meant we should change `default-deny` to "default deny" to indicate that's not some pre-built policy pattern, which I changed accordingly. Did I misunderstand your point here?

Reply:
Sorry, I wasn't clear. I suggested using "cluster-wide default-deny" as an example of a baseline policy, since currently our policy API is not rich enough to express "cross-namespace default-deny" in a simple way, unless I am mistaken.

Reply:
That is true. Our E2E testcase for baseline tier provides and example to default isolate a specific namespace: https://github.com/vmware-tanzu/antrea/blob/96fc15cbf5b33b3908163fcf837e2d2f3c83cf1a/test/e2e/antreapolicy_test.go#L328
but I agree that right now we don't have an easy way to specify a single policy that takes care of "cross-namespace default-deny" in the entire cluster. This would require some lambda sort of expressiveness in policy declaration. What about we change the doc to "baseline policies like default-deny cross-namespace traffic for specific namespaces" and provide the above ACNP as example?

Reply:
Updated